.github/workflows/code-review.yml

```
@@ -4,6 +4,10 @@ on:
     branches:
       - main
       - release-*
+    paths-ignore:
+      - "**.md"
+      - "**.yml"
```

why ignore only these two type of files and why all yml file in all subpaths?

i suggest to use an "allowed-list" strategy, so:
```suggestion
    paths:
      - "**/src/main/**"
      - "**pom.xml"
      - "Dockerfile"
      - ".github/workflows/code-review.yml"
```

---

fixed with https://github.com/pagopa/pdv-ms-user-registry/pull/19/commits/343cf355b24dcbc9b471f8838ebbdf0522957cf8